integration_test/third_party_apps_test/applications/couchdb/metadata.yaml

```diff
@@ -23,6 +23,35 @@ description: |-
   database metrics, such as how many are open and the number of operations. The
   integration collects general CouchDB and access logs and parses them into a JSON
   payload. The result includes fields for user, host, level, and message.
+configure_integration: |-
+  This integration does not require any additional configuration with
+  CouchDB versions prior to 3.4.x, or on Debian-based distributions.
+
+  As of version 3.4.0, CouchDB behaves differently on RPM-based
```

Comment:
The other files say 3.4.1. I may have mistyped it…

Reply:
Normalized everything to 3.4.0.